app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -98,11 +102,14 @@ def process_submission
 
           ::Rails.logger.info(post_create_log_msg(appeal_submission_id:, submitted_appeal_uuid:))
           if form4142.present?
-            handle_4142(request_body: req_body_obj,
-                        form4142:,
-                        appeal_submission_id:, submitted_appeal_uuid:)
+            handle_4142(request_body: req_body_obj, form4142:, appeal_submission_id:, submitted_appeal_uuid:)
+            uploaded_forms << '21-4142'
```

Comment:
Hmm since the 4142 upload is async, something we might consider is updating the SavedClaim in the job after the transaction is successful. But probably okay to leave this as-is